Packs/Base/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1

```diff
@@ -401,7 +482,7 @@ function ReturnOutputs([string]$ReadableOutput, [object]$Outputs, [object]$RawRe
         Type           = [EntryTypes]::note;
         ContentsFormat = [EntryFormats]::json.ToString();
         HumanReadable  = $ReadableOutput;
-        Contents       = $RawResponse;
+        Contents       = Remove-SelfReferences $RawResponse;
         EntryContext   = $Outputs
```

Comment:
Add flag to Remove-Self References in this function and add it to outputs as well (if flag is true).